first_plugin_ida/first.py

```diff
@@ -3113,10 +3117,7 @@ def __init__(self, dialog):
 
             def __data(self, thread, data):
                 if ('failed' in data) and data['failed']:
-                    if 'msg' not in data:
-                        return
-
-                    msg = '[1st] Error: {}'.format(data['msg'])
+                    msg = '[1st] Error: {}'.format(result['msg'])
```

Comment:
This will cause an exception to occur, result is not a variable in the scope of __data. I believe the original master and dev branch correctly checks.